spring-integration-debezium/src/main/java/org/springframework/integration/debezium/dsl/Debezium.java

```diff
@@ -0,0 +1,144 @@
+			Builder<ChangeEvent<byte[], byte[]>> debeziumEngineBuilder) {
+		return new DebeziumMessageProducerSpec(debeziumEngineBuilder);
+	}
+
+	/**
+	 * Create an instance of {@link DebeziumMessageProducerSpec} for the provided native debezium {@link Properties} and
+	 * JSON serialization formats.
+	 * @param debeziumConfig {@link Properties} with required debezium engine and connector properties.
+	 * @return the spec.
+	 */
+	public static DebeziumMessageProducerSpec inboundChannelAdapter(Properties debeziumConfig) {
+
+		return new DebeziumMessageProducerSpec(builder(
+				debeziumConfig, io.debezium.engine.format.JsonByteArray.class,
+				io.debezium.engine.format.JsonByteArray.class, null, null,
+				null, null));
+	}
+
+	/**
+	 * Create an instance of {@link DebeziumMessageProducerSpec} for the provided native debezium {@link Properties} and
+	 * serialization formats.
+	 * @param debeziumConfig {@link Properties} with required debezium engine and connector properties.
+	 * @param messageFormat {@link SerializationFormat} format for the {@link ChangeEvent} key and payload.
+	 * @param headerFormat {@link SerializationFormat} format for the {@link ChangeEvent} headers.
+	 * @return the spec.
+	 */
+	public static DebeziumMessageProducerSpec inboundChannelAdapter(Properties debeziumConfig,
+			Class<? extends SerializationFormat<byte[]>> messageFormat,
+			Class<? extends SerializationFormat<byte[]>> headerFormat) {
+
+		return new DebeziumMessageProducerSpec(builder(
+				debeziumConfig, messageFormat, headerFormat, null, null, null, null));
+	}
+
+	/**
+	 * Create an instance of {@link DebeziumMessageProducerSpec} from provided debezium configuration.
+	 * @param debeziumConfig {@link Properties} with required debezium engine and connector properties.
+	 * @param messageFormat {@link SerializationFormat} format for the {@link ChangeEvent} key and payload.
+	 * @param headerFormat {@link SerializationFormat} format for the {@link ChangeEvent} headers.
+	 * @param offsetCommitPolicy Set the Debezium commit policy. The default is a periodic commit policy based on time *
+	 * intervals.
+	 * @param completionCallback Set the completion callback that logs the completion status.
+	 * @param connectorCallback Set a callback to provide feedback about the different stages according to the
+	 * completion state of each component running within the engine.
+	 * @param debeziumClock Set a specific clock when needing to determine the current time. Defaults to
+	 * {@link Clock#systemDefaultZone() system clock}
+	 * @return the spec.
+	 */
+	public static DebeziumMessageProducerSpec inboundChannelAdapter(Properties debeziumConfig,
+			Class<? extends SerializationFormat<byte[]>> messageFormat,
+			Class<? extends SerializationFormat<byte[]>> headerFormat,
+			@Nullable OffsetCommitPolicy offsetCommitPolicy, @Nullable CompletionCallback completionCallback,
+			@Nullable ConnectorCallback connectorCallback, @Nullable Clock debeziumClock) {
```

Comment:
This is too much for a single factory method.
Better to point to the method based on a builder if there is need for more sophisticated configuration.
My point was only about simple use-cases where we have props and are OK with the rest as defaults.

So, why do you introduce this complex method?